interface_config.js

```diff
@@ -183,15 +187,21 @@ var interfaceConfig = {
      */
     SUPPORT_URL: 'https://community.jitsi.org/',
 
-    TOOLBAR_ALWAYS_VISIBLE: false,
+    /**
+     * DEPRECATED! Moved to config.js as `toolbarConfig.alwaysVisible`.
+     */
+    // TOOLBAR_ALWAYS_VISIBLE: false,
 
     /**
      * DEPRECATED!
      * This config was moved to config.js as `toolbarButtons`.
      */
     // TOOLBAR_BUTTONS: [],
 
-    TOOLBAR_TIMEOUT: 4000,
+    /**
```

Comment:
Can you group all these deprecated toolbox options together in a single large(r) comment? This way it's easier to spot at a glance.